### PR TITLE
AK: Handle edge cases correctly in hypot

### PR DESCRIPTION
**Pull request by LucasChollet**
Fixes this libcxx test:
std/numerics/complex.number/complex.value.ops/abs.pass.cpp

(`abs` is implemented as `hypot(real, imag)`)

### CODE REVIEW

File: Tests/LibC/TestMath.cpp [hunk 1]
```diff
@@ -395,3 +395,20 @@ TEST_CASE(round)
     EXPECT_EQ(round(9999999999999.5), 10000000000000.0);
     EXPECT_EQ(round(-9999999999999.5), -10000000000000.0);
 }
+
+TEST_CASE(hypot)
```
Comment (Hendiadyoin1) [inline Tests/LibC/TestMath.cpp:399]: Maybe also add a test for (inf, nan)/ change the (inf,inf) test to that, as that's the actual edge case
